src/lib/hooks/useSideMenu.js

```diff
@@ -126,10 +127,15 @@ export default (props = {}) => {
         centered: true,
         name: 'Statistics',
         action: () => {
-          navigation.navigate({
-            routeName: 'Statistics',
-            type: 'Navigation/NAVIGATE',
-          })
+          if (isWeb && (isBrowser || isTablet)) {
```

Comment:
1. add isPhone flag to the utils/platform (which detects is the device an phone or iPod-like player exactly)

then
```suggestion
          if (isWeb && !isPhone) {
```

2. use dashboardLink from the Config directly please. don't touch vewViewInstances, the constants defined inside are just for the internal usage

3. import openLink from src/lib/utils/linking.js and use it instead of window.open. You could also specify '_blank' as the second param to open it into a new tab